x-pack/plugins/security_solution/public/common/components/callouts/callout.tsx

```diff
@@ -36,8 +37,10 @@ const CallOutComponent: FC<CallOutProps> = ({
       data-test-subj={`callout-${id}`}
       data-test-messages={`[${id}]`}
     >
-      <CallOutDescription messages={message} />
-      <CallOutDismissButton message={message} text={dismissButtonText} onClick={onDismiss} />
+      {description}
```

Comment:
NOTE: This is where I just use `description` directly and have deleted the `<CallOutDescription messages={message} />` since that's not being used anywhere else and this does not have the capability to send down arrays into it. Seemed better to have less code by directly just using descrption.

Reply:
I'm ok with that 👍